app/controllers/support/users_controller.rb

```diff
@@ -22,6 +22,8 @@ def edit
 
     def create
       @user = User.new(user_params)
+      @user.email.strip!
```

Comment:
There is a `downcase_email` in a `before_save` hook on the `User` model. My preference would be to have both the strip and the downcase happen at the same time, and on the model. There may be other methods for creating a user and we will always want the whitespace removed. 

What do you think?

Reply:
Sweet!

Reply:
Yup defo agree, updated here: https://github.com/DFE-Digital/publish-teacher-training/pull/4112/commits/3661e071d72c5723c631c94002bdb93dd52d2711